### DISCUSSION TITLE
Show Output as a tab (the way it was before)

### DISCUSSION THREAD

**Discussion by Moridin11** [dbeaver/dbeaver#11601] [Q&A] [answered]
The output  used to show as a tab next to the results window.  Is there a way for us to move it back there. After updating to 7.3.4 it displays to the right of the SQL Query and this is rather annoying.

Was this intended or is this something that we can be given the option?

**Comment by ShadelessFox** [comment, marked as answer]
Hello @Moridin11,

This change was intentional and you cannot move it back like it was before.

You can disable automatic opening using <kbd>Preferences</kbd> &rArr; <kbd>Editors</kbd> &rArr; <kbd>SQL Editor</kbd> &rArr; <kbd>Open output viewer on new messages</kbd>
